src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1208,4 +1209,36 @@ export class NetworkAreaDiagramViewer {
             }
         }
     }
+
+    private updateMetadataCallCallback(mousePosition: Point) {
+        // get move node in metadata
+        const node: SVGGraphicsElement | null = this.container.querySelector(
+            'nad\\:node[svgid="' + this.selectedElement?.id + '"]'
+        );
+        if (node != null) {
+            // save original position, for the callback
+            const xOrig = node.getAttribute('x') ?? 0;
+            const yOrig = node.getAttribute('y') ?? 0;
+            // update node position in metadata
+            node.setAttribute(
+                'x',
+                DiagramUtils.getFormattedValue(mousePosition.x)
+            );
+            node.setAttribute(
+                'y',
+                DiagramUtils.getFormattedValue(mousePosition.y)
+            );
+            // call the callback, if defined
+            if (this.onNodeCallback != null) {
+                this.onNodeCallback(
+                    node.getAttribute('equipmentid') ?? '',
+                    node.getAttribute('svgid') ?? '',
+                    DiagramUtils.round(mousePosition.x, 2),
```

Comment:
You'd rather reuse the exact same value as in the metadata. Just introduce a variable `const xNew = DiagramUtils.getFormattedValue(mousePosition.x)`, and similarly for y.

Reply:
done